packages/categorize/configure/src/design/choices/choice.jsx

```diff
@@ -63,8 +64,13 @@ export class Choice extends React.Component {
     onChange(choice);
   };
 
+  isCheckboxShown = allowMultiplePlacements => {
+    return !(allowMultiplePlacements === 'Yes' || allowMultiplePlacements === 'No');
```

Comment:
Wouldn't this be more simple if you check allowMultiplePlacements === 'Set Per Choice' ? 
Also, maybe you can create a constant with these values 'Yes', 'No', 'Set Per Choice' since they are used multiple times. Something like
 `multiplePlacements: { enabled: 'Yes', disabled: 'No', perChoice: 'Set Per Choice' }`